docs/modules/ROOT/pages/erc721.adoc

```diff
@@ -269,10 +246,9 @@ mod MyTokenReceiver {
 
 === Storing ERC721 URIs
 
-:string-roadmap: https://github.com/orgs/starkware-libs/projects/1/views/1?pane=issue&itemId=28823165[here]
-
-Token URIs in Cairo are stored as single field elements (`felt252`).
-Each field element equates to 252-bits (or 31.5 bytes) which means that a token's URI can be no longer than 31 characters.
+:solidity-impl: https://github.com/OpenZeppelin/openzeppelin-contracts/blob/932fddf69a699a9a80fd2396fd1a2ab91cdda123/contracts/token/ERC721/ERC721.sol#L85-L93[Solidity implementation]
+:token-uri: xref:/api/erc721.adoc#IERC721Metadata-token_uri[token_uri]
 
-NOTE: Native string support in Cairo is currently in progress and tracked {string-roadmap}.
-Once Cairo offers full string support, this will be revisited.
+Token URIs were previously stored as single field elements prior to Cairo v0.2.5.
```

Comment:
should we mention the URI storage strategy (base + concatenation) in here?

Reply:
The next line describes the concatenation, no? Or do you mean the URIStorage extension that we want to add in the future?

Reply:
yes i think i was looking at just part of the diff, didn't see that part